### PR TITLE
brigand: New Package

### PR DESCRIPTION
**Pull request by nilsdeppe**

### CODE REVIEW

File: var/spack/repos/builtin/packages/brigand/package.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    version('master', git='https://github.com/edouarda/brigand.git',
+            branch='master')
+    version('1.3.0', '0bea9713b3b712229aed289e218d577b')
+    version('1.2.0', '32c0f73e7e666d33ff123334f5c9c92f')
+    version('1.1.0', '073b7c8e2cbda3a81bbeb1ea5b9ca0eb')
+    version('1.0.0', 'eeab3d437090f0bb7bc4eb69a5cd9c49')
+
+    def install(self, spec, prefix):
+        import shutil
+        shutil.copytree('include', prefix.include)
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/brigand/package.py]: You should be able to use:
```python
install_tree('include', prefix.include)
```
without importing anything.

File: var/spack/repos/builtin/packages/brigand/package.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        # fakes out spack so it installs a module file
+        mkdirp(join_path(prefix, 'bin'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/brigand/package.py]: Does Spack not write a module file unless prefix.bin exists??

**Comment by nilsdeppe** [comment]
Thanks for pointing out those two things! The module part came from the `catch` spack repo, so it might not be needed there anymore either.

Should the `install_tree` command be added to the docs?

**Comment by adamjstewart** [comment]
> Should the `install_tree` command be added to the docs?

Probably. There's a lot of things that should be added to the docs. If you have the time, feel free to submit a PR. This command is defined [here](https://github.com/LLNL/spack/blob/develop/lib/spack/llnl/util/filesystem.py#L216,L223).

**Comment by nilsdeppe** [comment]
Alright, I'll take a look at adding that to the docs this week. Thanks!

**Comment by davydden** [comment]
@nilsdeppe then you can mark this issues as resolved https://github.com/LLNL/spack/issues/1581

**Pull request merged**